src/sheet.tsx

```diff
@@ -531,10 +533,12 @@ const BaseSheet = forwardRef<HTMLDivElement, InteralSheetProps>(
             ...modal
           }}
         >
-          <div
-            className={cx(`${prefix}-backdrop`, `${prefix}-stack`)}
-            {...bindEvents({ closeOnTap: true })}
-          ></div>
+          {showBackDrop && (
+            <div
+              className={cx(`${prefix}-backdrop`, `${prefix}-stack`)}
+              {...bindEvents({ closeOnTap: true })}
+            ></div>
+          )}
           <TrapFocus>
```

Comment:
If `showBackDrop` is enabled, the TrapFocus component should not be applied to allow keyboard nav behind the dialog